lib/gems/pending/VMwareWebService/MiqVimDataStore.rb

```diff
@@ -287,7 +286,7 @@ def get_file_content(filepath)
     /(\d*)\.(\d*)/ =~ @invObj.about['version']
     raise "get_file_content not supported on [#{@invObj.about['fullName']}]" if $1.to_i < 3 || ($1.to_i == 3 && $2.to_i < 5)
 
-    fileUrl = "https://#{@invObj.server}/folder/#{MIQEncode.base64Encode(filepath)}?dsName=#{MIQEncode.base64Encode(@name)}"
+    fileUrl = "https://#{@invObj.server}/folder/#{Base64.encode64(filepath)}?dsName=#{Base64.encode64(@name)}"
```

Comment:
Gah you're right, badly named method :(
What do you think about removing `base64Encode` from `MIQEncode` then?  I don't see any other callers of it [ref](https://github.com/search?utf8=%E2%9C%93&q=org%3AManageIQ+MIQEncode.base64Encode&type=Code) [ref](https://github.com/search?utf8=%E2%9C%93&q=org%3AManageIQ+MIQEncode.base64Decode&type=Code)